recipes/crlibm/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "crlibm" %}
+{% set version = "1.0b5" %}
```

Comment:
Is there a stable, non-beta version of this package?

Reply:
Nope.

Reply:
There isn't even an official repo. That's why I was hesitant to split them up in the first place.

Reply:
@scopatz Is this a blocking issue?

Reply:
We actually just approved changes allowing these kinds of packages to go in, but I think the recipe will need to be slightly different.  Please see https://github.com/conda-forge/cfep/blob/master/cfep-05.md

If you have any questions please ping @conda-forge/core or ask on Gitter

Reply:
@scopatz, this package is no longer maintained, so there's no point it putting the package into a dev label.

Reply:
If I wasn't urged to go down this rabbit hole we would be happily referring to a stable version of a pypi package now :-)

Reply:
Hmm OK, I won't block this then!

Reply:
@scopatz Thank you!